examples/showcase/bar/bar-export-2/demo.details

```diff
@@ -0,0 +1,7 @@
+---
+ name: Bar Chart with custom data labels
+ description: Here we've created a custom visualization that puts the data labels at the end of the chart align with each bar, or at the end of the bar.
```

Comment:
nit: align -> aligned
question: which side is the "end of the chart"? since it's a bar chart, do you always mean the right?

Reply:
Updated the text slightly.